crates/rattler_conda_types/src/version/mod.rs

```diff
@@ -634,17 +720,17 @@ mod test {
         assert_eq!(random_versions, parsed_versions);
     }
 
-    #[test]
-    fn bump() {
-        assert_eq!(
-            Version::from_str("1.1").unwrap().bump(),
-            Version::from_str("1.2").unwrap()
-        );
-        assert_eq!(
-            Version::from_str("1.1l").unwrap().bump(),
-            Version::from_str("1.2l").unwrap()
-        )
-    }
+    // #[test]
```

Comment:
Are we ready to re-enable this test? Or do we not need that function anymore?

Reply:
Btw. in `rattler-build` we have some functions like `pin_compatible(otherpkg, min_pin="x.x", max_pin="x.x")` which would resolve to `>=1.4,<1.5`, so being able to "bump" versions is nice.